### PR TITLE
Update the main page style

### PR DESCRIPTION
**Pull request by clyang82**
Signed-off-by: clyang82 <chuyang@redhat.com>

### CODE REVIEW

**Comment by clyang82** [comment]
/hold
will update the png later.

File: themes/ocmTheme/static/homepage.css [hunk 1]
```diff
@@ -60,8 +76,34 @@
...
+    height: calc(100% + 2em);
+  }
+}
+@media (max-width: 992px) {
+  .subject-img {
+    width: calc(100% + (992px - 720px)/2 + 16px);
+    height: calc(100% + 2em);
+  }
+}
+@media (max-width: 768px) {
```
Comment (skeeey) [inline themes/ocmTheme/static/homepage.css:102]: we may not need this

File: themes/ocmTheme/layouts/partials/subject.html [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
         </div>
       </div>
-      <img src="/robot7.png" alt="" class="subject-img subject-img-none" />
+      <div class="col-8 mx-auto col-md-4 order-md-2 col-lg-7">
+        <img src="/robot7.png" alt="" class="subject-img subject-img-none" />
+      </div>
     </div>
   </div>
-</section>
+</div>
```
Comment (skeeey) [inline themes/ocmTheme/layouts/partials/subject.html:19]: need a new line

File: themes/ocmTheme/layouts/partials/header.html [hunk 3]
```diff
@@ -1,56 +1,58 @@
...
+              <i class="bi bi-slack"></i>
+            </a>
+          </li>
+        </ul>
+      </div>
     </div>
-  </div>
-</nav>
+  </nav>
+</header>
```
Comment (skeeey) [inline themes/ocmTheme/layouts/partials/header.html:58]: need a new line

**Comment by skeeey** [comment]
/lgtm

@qiujian16 would you help merge this pr, this pr is used to enhance the main page responsive style

**Pull request merged**